lib/libutils/sub.mk

```diff
@@ -1,2 +1,3 @@
 subdirs-$(CFG_LIBUTILS_WITH_ISOC) += isoc
 subdirs-y += ext
+subdirs-y += compiler-rt
```

Comment:
Should we add a `$(CFG_LIBUTILS_WITH_COMPILER_RT)`?

Reply:
Preferably, yes. Although I doubt it will often be disabled, at least this is consistent with `CFG_LIBUTILS_WITH_ISOC` and it is also an opportunity to document in `mk/config.mk` what purpose compiler-rt serves.